ci/docker/runtime_functions.sh

```diff
@@ -806,6 +806,24 @@ build_ubuntu_gpu_cuda101_cudnn7() {
     make cython PYTHON=python3
 }
 
+build_ubuntu_gpu_cuda110_cudnn8() {
+    set -ex
+    build_ccache_wrappers
+    make \
+        USE_BLAS=openblas                         \
+        USE_MKLDNN=0                              \
+        USE_CUDA=1                                \
+        USE_CUDA_PATH=/usr/local/cuda             \
+        USE_CUDNN=1                               \
+        USE_TVM_OP=0                              \
+        USE_CPP_PACKAGE=1                         \
+        USE_DIST_KVSTORE=1                        \
+        CUDA_ARCH="$CI_CUDA_COMPUTE_CAPABILITIES" \
```

Comment:
I notice that earlier in this file, we have the lines
```
CI_CUDA_COMPUTE_CAPABILITIES="-gencode=arch=compute_52,code=sm_52 -gencode=arch=compute_70,code=sm_70"
CI_CMAKE_CUDA_ARCH="5.2 7.0"
```
Not sure what we're trying to prove exactly with this PR, but do we gain any confidence by adding arch 80, even though there are no A100's in the CI?

Reply:
👍 can we do this @josephevans ?

Reply:
I think the team just wanted to know it would build and test against cuda 11.0 and libcudnn8, which is what this PR tries to address. We could add arch 80 for just this build.

Reply:
Added compute_80 and sm_80 to CUDA_ARCH.